tests/storage/datavolume.go

```diff
@@ -1063,6 +1064,11 @@ var _ = SIGDescribe("DataVolume Integration", func() {
 			dataVolume, err = virtClient.CdiClient().CdiV1beta1().DataVolumes(testsuite.GetTestNamespace(nil)).Create(context.Background(), dataVolume, metav1.CreateOptions{})
 			Expect(err).ToNot(HaveOccurred())
 
+			// Importing Fedora is so slow that we get "resourceVersion too old" when trying
+			// to watch for events between the VMI creation and VMI starting.
+			By("Making sure the slow Fedora import is complete before creating the VMI")
+			libstorage.EventuallyDV(dataVolume, 500, HaveSucceeded())
+
 			vmi = tests.RunVMIAndExpectLaunchWithDataVolume(vmi, dataVolume, 500)
```

Comment:
Maybe worth to integrate this logic into RunVMIAndExpectLaunchWithDataVolume? We may have more tests that do slow Fedora imports

Reply:
I had to use immediate binding mode to be able to finish the DV import before the VMI is started, might not be good to do this generically.

I can also watch for events without resourceVersion and instead use:
```
-       objectEventWatcher := watcher.New(vmi).SinceWatchedObjectResourceVersion().Timeout(time.Duration(seconds+2) * time.Second)
+       objectEventWatcher := watcher.New(vmi).SinceNow().Timeout(time.Duration(seconds+2) * time.Second)
```
but this means missing warning events from when the VMI is first started.

I don't think these generalizations would be acceptable

Reply:
Yeah don't want to go there, should probably just merge this and not panic over the resourceVersion too old err in the other PR